### PR TITLE
Add repl deploy to release workflow

### PR DESCRIPTION
**Pull request by devongovett**
Copied from the ci workflow but using `--prod` instead of a PR alias domain

### CODE REVIEW

File: .github/workflows/tag-release.yml [hunk 1]
```diff
@@ -216,3 +216,75 @@ jobs:
...
+      #     name: REPL
+      #     path: 'packages/dev/repl/dist'
+      - name: Start Deployment
+        uses: bobheadxi/deployments@v1
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.head_ref }}
+          env: Preview
```
Comment (mischnic) [inline .github/workflows/tag-release.yml]: We should make that something like `Production`, though this isn't really exposed that much in the Github UI: https://github.com/parcel-bundler/parcel/deployments

File: .github/workflows/tag-release.yml [hunk 2]
```diff
@@ -216,3 +216,75 @@ jobs:
...
+          github-comment: false
+          working-directory: packages/dev/repl
+          vercel-args: '--prod'
+      - name: Update Deployment Status
+        uses: bobheadxi/deployments@v1
+        if: always()
+        with:
+          step: finish
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: Preview
```
Comment (mischnic) [inline .github/workflows/tag-release.yml]:
```suggestion
          env: Production
          override: false
      - name: Deploy to Vercel
        uses: amondnet/vercel-action@v25
        id: vercel-action
        with:
          vercel-token: ${{ secrets.REPL_VERCEL_TOKEN }}
          vercel-org-id: ${{ secrets.REPL_VERCEL_ORG_ID }}
          vercel-project-id: ${{ secrets.REPL_VERCEL_PROJECT_ID }}
          github-comment: false
          working-directory: packages/dev/repl
          vercel-args: '--prod'
      - name: Update Deployment Status
        uses: bobheadxi/deployments@v1
        if: always()
        with:
          step: finish
          token: ${{ secrets.GITHUB_TOKEN }}
          env: Production
```

**Review by mischnic — APPROVED** [review]

File: .github/workflows/tag-release.yml [hunk 3]
```diff
@@ -216,3 +216,75 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: yarn release
+
+  # Deployment steps taken from https://github.com/colinwilson/static-site-to-vercel/blob/master/.github/workflows/deploy-preview.yml
+  repl_build:
+    name: Build REPL
+    runs-on: ubuntu-latest
+    needs: [unit_tests]
```
Comment (MonicaOlejniczak) [inline .github/workflows/tag-release.yml]: Will this work cross-workflow?

File: .github/workflows/tag-release.yml [hunk 4]
```diff
@@ -216,3 +216,75 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: yarn release
+
+  # Deployment steps taken from https://github.com/colinwilson/static-site-to-vercel/blob/master/.github/workflows/deploy-preview.yml
+  repl_build:
```
Comment (MonicaOlejniczak) [inline .github/workflows/tag-release.yml:221]: How come this isn't in the nightly workflow as well?

Comment (devongovett) [reply .github/workflows/tag-release.yml:221]: probably don't want to deploy to production on every nightly, just when we do stable releases.

**Pull request merged**